### DISCUSSION TITLE
How to list issues in a current sprint?

### DISCUSSION THREAD

**Discussion by sarmong** [ankitpokhrel/jira-cli#725] [Q&A] [answered]
How to list issues in a current (or any for that matter) sprint?

**Comment by ankitpokhrel** [comment, marked as answer]
You can do it as follows:

```sh
# List issues in current sprint
$ jira sprint list --current

# List issues in sprint with id 9
$ jira sprint list 9
```

**Reply by aniketgm** [reply]
Hey @ankitpokhrel, I have a similar question related to sprint. I want to list the issues in current sprint as well however with only the assignee as me. I see no option under `jira sprint list` to filter by assignee. So, I tried the following JQL but doesn't seem to work.
I'm still learning JQL. Can you pls help with this? What am I missing?
```sh
# I tried this 
jira sprint list --current -q "assignee is 'Aniket Meshram'"
# And also this:
jira sprint list --current -q "assignee == 'Aniket Meshram'"
# And also this with == and 'is'
jira sprint list --current -q "assignee == '<email-id>'"
```

Or is there any way to filter issues by sprint from the issues command, where, for e.g.:
```sh
jira issues list --assignee "Aniket Meshram" -q "<jql-query>" # OR an option like: --sprint <sprint-id>
```

**Reply by ankitpokhrel** [reply]
Hi @aniketgm, correct jql would be something like this:

```sh
$ jira sprint list --current -q 'assignee="email@example.com"'
$ jira sprint list --current -q "assignee='Aniket Meshram'"
$ jira sprint list --current -q 'assignee=Aniket'
```
